src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1652,7 +1652,7 @@ public T onCompleted(Response response) throws Exception {
                 if (response.getStatusCode() == 404) {
                     return null;
```

Comment:
Convention is to return `null` when there's a 404, like here. The new exception will be not thrown here.